### ISSUE TITLE
Unexpected error in tidyr::fill

### ISSUE THREAD

**Issue by cboettig** [duckdb/duckdb#2372]
**What does happen?**

Hi @Mytherin & @mgirlich, sorry, here's another case where dbplyr's SQL runs on SQLite but not duckdb, once again not sure if this one is an issue on duckdb side or the dbplyr side.  Looks like something to do with `IGNORE NULL`:

The following example comes from the docs of `dblpyr::fill()`, and runs fine on SQLite but fails on duckdb connection:

```r
squirrels <- tibble::tribble(
  ~group,    ~name,     ~role,     ~n_squirrels, ~ n_squirrels2,
  1,      "Sam",    "Observer",   NA,                 1,
  1,     "Mara", "Scorekeeper",    8,                NA,
  1,    "Jesse",    "Observer",   NA,                NA,
  1,      "Tom",    "Observer",   NA,                 4,
  2,     "Mike",    "Observer",   NA,                NA,
  2,  "Rachael",    "Observer",   NA,                 6,
  2,  "Sydekea", "Scorekeeper",   14,                NA,
  2, "Gabriela",    "Observer",   NA,                NA,
  3,  "Derrick",    "Observer",   NA,                NA,
  3,     "Kara", "Scorekeeper",    9,                 10,
  3,    "Emily",    "Observer",   NA,                NA,
  3, "Danielle",    "Observer",   NA,                NA
)
squirrels$id <- 1:12

con <- DBI::dbConnect(duckdb::duckdb())

## SQLite is happy:
#con <- DBI::dbConnect(RSQLite::SQLite())

dbWriteTable(con, "squirrels", squirrels, overwrite=TRUE)

## errors
tbl(con, "squirrels") %>%
    window_order(id) %>%
    tidyr::fill(
      n_squirrels,
      n_squirrels2,
    )
``` 

Here's the error trace  I see:

```

Error in .local(conn, statement, ...) : 
  duckdb_prepare_R: Failed to prepare query SELECT "group", "name", "role", "id", LAST_VALUE("n_squirrels" IGNORE NULLS) OVER (ORDER BY "id") AS "n_squirrels", LAST_VALUE("n_squirrels2" IGNORE NULLS) OVER (ORDER BY "id") AS "n_squirrels2"
FROM "squirrels"
LIMIT 11
Error: Parser Error: syntax error at or near "IGNORE"
LINE 1: ..."role", "id", LAST_VALUE("n_squirrels" IGNORE NULLS) OVER (ORDER BY "id") AS "...
```

**What should happen?**
Describe what you expect DuckDB to do instead.

Query results should match those from the SQLite driver, e.g. 

```r
# Source:     lazy query [?? x 6]
# Database:   sqlite 3.36.0 []
# Ordered by: id
   group name     role        n_squirrels n_squirrels2    id
   <dbl> <chr>    <chr>             <dbl>        <dbl> <int>
 1     1 Sam      Observer             NA            1     1
 2     1 Mara     Scorekeeper           8            1     2
 3     1 Jesse    Observer              8            1     3
 4     1 Tom      Observer              8            4     4
 5     2 Mike     Observer              8            4     5
 6     2 Rachael  Observer              8            6     6
 7     2 Sydekea  Scorekeeper          14            6     7
 8     2 Gabriela Observer             14            6     8
 9     3 Derrick  Observer             14            6     9
10     3 Kara     Scorekeeper           9           10    10
# … with more rows
```

**To Reproduce**
See reprex above

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - DuckDB Version current 0.2.9, and master branch

**Before submitting**
- [x] Have you tried the steps to reproduce? Do they include all relevant data and configuration? Does the issue you report still appear there?
- [x] Have you tried this on the latest `master` branch? In case you cannot compile, you may find some binaries here: https://github.com/duckdb/duckdb/releases/tag/master-builds

**Comment by hawkfish** [comment]
It looks like `dblpyr` is not using Postgres for DuckDB, although there is code to handle databases that don't support `IGNORE NULLS` [here](https://github.com/tidyverse/dbplyr/blob/1b7c77ef921925d7f9875200174c7f9bb28fc503/R/verb-fill.R#L123). We probably need to set up an alias somehow.

**Comment by mgirlich** [comment]
Currently, there are no extra code paths for DuckDB in `dbplyr`. I assume we can support + test DuckDB in `dbplyr`. To fix this issue it should be sufficient to add the following line

```r
dbplyr_fill0.duckdb_connection <- dbplyr_fill0.SQLiteConnection
```

I created an issue https://github.com/tidyverse/dbplyr/issues/714 for the support. If you want to support the work feel free to test other `dbplyr` functions with DuckDB and add a note whether they work or not.

**Comment by cboettig** [comment]
Looks like this is fixed in duckdb 0.3.3, thanks!
